mysql/resource_rds_config_test.go

```diff
@@ -0,0 +1,206 @@
+package mysql
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"log"
+	"strconv"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+func TestAccResourceRDS(t *testing.T) {
+	rName := acctest.RandStringFromCharSet(10, acctest.CharSetAlpha)
```

Comment:
Please don't use randomness in tests. That way, tests crash randomly and it's difficult to find the reason.

If you want to test more inputs, do it by using table-based tests.

Reply:
OK, will remove randomness.